C/plugins/north/OMF/plugin.cpp

```diff
@@ -1553,42 +1548,145 @@ int PIWebAPIGetVersion(CONNECTOR_INFO* connInfo, bool logMessage)
 	_PIWebAPI->setAuthMethod          (connInfo->PIWebAPIAuthMethod);
 	_PIWebAPI->setAuthBasicCredentials(connInfo->PIWebAPICredentials);
 
-	int httpCode = _PIWebAPI->GetVersion(connInfo->hostAndPort, connInfo->PIWebAPIVersion, logMessage);
+	int httpCode = _PIWebAPI->GetVersion(connInfo->hostAndPort, connInfo->RestServerVersion, logMessage);
 	delete _PIWebAPI;
 
 	return httpCode;
 }
 
+static std::string ParseEDSProductInformation(std::string json)
```

Comment:
We should have doxygen comment blocks on new functions.

Reply:
Done. I also created a new routine called _ParseProductVersion_ so that the version parsing is not done directly in _SetOMFVersion_.